Makefile

```diff
@@ -201,9 +201,11 @@ test-ci-js:
 		$(TEST_CI_ARGS) $(CI_JS_SUITES)
 
 test-ci: | build-addons
-	$(PYTHON) tools/test.py $(PARALLEL_ARGS) -p tap --logfile test.tap \
+	out/Release/cctest --gtest_output=tap:cctest.tap
+	$(PYTHON) tools/test.py $(PARALLEL_ARGS) -p tap --logfile jstest.tap \
 		--mode=release --flaky-tests=$(FLAKY_TESTS) \
 		$(TEST_CI_ARGS) $(CI_NATIVE_SUITES) $(CI_JS_SUITES)
+	$(PYTHON) tools/merge-tap.py test.tap cctest.tap jstest.tap
```

Comment:
Do we really have to merge these? We can just tell Jenkins to pick all three up.